src/apis/vaults.ts

```diff
@@ -296,40 +298,44 @@ export class DefaultVaultsAPI {
         return await this.api.query.vaultRegistry.premiumRedeemThreshold();
     }
 
-    async getFees(_vaultId: AccountId): Promise<PolkaBTC> {
-        // TODO: get real value from backend
-        return this.api.createType("FixedU128", 368) as PolkaBTC;
+    async getFees(vaultId: string): Promise<string> {
+        // TODO: integration test using docker-compose setup
+        const parsedId = this.api.createType("AccountId", vaultId);
+        return (await this.api.query.fee.totalRewards(parsedId)).toString();
     }
 
-    async getAPY(vaultId: AccountId): Promise<string> {
+    async getAPY(vaultId: string): Promise<string> {
+        // TODO: integration test using docker-compose setup
         const fees = await this.getFees(vaultId);
-        // TODO: convert from PolkaBTC to USD
-        const feesInUSD = fees;
-
-        const lockedDot = await this.collateralAPI.balanceLockedDOT(vaultId);
-        // TODO convert from DOT to USD
-        const lockedUsdValue = lockedDot;
-
-        return feesInUSD.div(lockedUsdValue).toString();
+        const feesBig = new Big(fees.toString());
+        const dotToBtcRate = await this.oracleAPI.getExchangeRate();
+        const feesInDot = feesBig.div(new Big(dotToBtcRate));
```

Comment:
You need to multiply here. If the fees were e.g. 1 PolkaBTC and the exchange rate is e.g. 3855.55 DOT/BTC you'd ned to multiply to get the DOT amount.

Reply:
I'll update, but if we're using DOT/BTC should the rate be something small, like `0.00026455` (https://uk.tradingview.com/symbols/DOTBTC/)?

Reply:
sorry, the other way round. We have BTC to DOT which is set to something like 3855.55. And then you need to multiply to get from BTC to DOT.

Reply:
Since it is the other way around, we should change the feed name returned by the client: https://github.com/interlay/polkabtc-js/blob/dev/test/unit/apis/oracle.test.ts#L58